V-spring-boot-app

VaolEr opened the pull request:
GS-20. Add more test data to tables.

VaolEr committed:
add log file

VaolEr committed:
GS-20. Add more test data to tables

dimio commented on src/main/resources/db/changelog/db.changelog-master.yaml:
```diff
@@ -10,4 +10,7 @@ databaseChangeLog:
-      file: db/changelog/data/2020-12-20-update-items-storehouses-ids.sql
\ No newline at end of file
+      file: db/changelog/data/2020-12-20-update-items-storehouses-ids.sql
+  # Increment created PK id, from 0 to 999
```
Комментарий поменяй, что мы дополняем БД данными тут

VaolEr replied:
Согласен, копипаста

dimio commented on src/main/resources/db/changelog/data/2020-12-21-add-some-data-to-tables.sql:
```diff
@@ -0,0 +1,194 @@
+
+
+
+-- changeset Valentin:Populate_items_storehouses_table
```
И здесь бы написал тогда, что добавляем на склады. Типа Add_items_quantity_to_storehouses

VaolEr replied:
Подправил.

VaolEr committed:
GS-20. Add more test data to tables;
Change log comments;
GS-21. Add indexes to all  fields 'name' in each table and unique index in items table for supplier_id and sku;

VaolEr committed:
GS-20. Add more test data to tables;
Change log comments;
GS-21. Add indexes to all  fields 'name' in each table and unique index in items table for supplier_id and sku;
